static/css/thing.css

```diff
@@ -374,12 +374,13 @@
 }
 
 .media-modal {
-  width: calc(100% - 15.2rem);;
+  width: calc(100% - 15.2rem);
```

Comment:
Given that our CSS files tend to be scoped to certain pages, it doesn't make a lot of sense to have the multi-use model stuff in `thing.css`. Could you instead lift it all into `app.css`?

Reply:
You mean ALL the modal things, including `media-modal`? So that it stays together?

Reply:
Yes.